scenarios/turbine-heat-sensor/gradle.properties

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+### dependencies
+junitVersion=4.11
+commonsCLIVersion=1.4
+pravegaVersion=0.3.0-50.f56b52d-SNAPSHOT
```

Comment:
do we need a separate file at the module level? Can we not make use of root gradle.properties configuration?

Reply:
It has been removed (see previous comment on this issue).